bigbluebutton-html5/imports/ui/components/actions-bar/create-breakout-room/component.jsx

```diff
@@ -383,6 +383,7 @@ class BreakoutRoom extends Component {
       valid,
       numberOfRooms,
     } = this.state;
+    const rooms = (numberOfRooms > MAX_BREAKOUT_ROOMS) ? 0 : numberOfRooms;
```

Comment:
You should check for MIN_BREAKOUT_ROOMS too, because it's possible to put a negative number